README.md

```diff
@@ -10,43 +10,12 @@ Odds are good that you don't want to invoke Ignition directly. In fact, it isn't
 
 - Mailing list: [coreos@lists.fedoraproject.org](https://lists.fedoraproject.org/archives/list/coreos@lists.fedoraproject.org/)
 - IRC: #[fedora-coreos](irc://irc.freenode.org:6697/#fedora-coreos) on freenode.org
-- Bugs: [issues][issues]
+- Reporting bugs: [issues](https://github.com/coreos/ignition/issues/new/choose)
 
 ## Contributing
 
 See [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches and the contribution workflow.
 
 To help triage or fix bugs, see the current [Ignition issues](https://github.com/coreos/ignition/issues/).
 
-## Reporting Bugs
-
-- To report a bug, use the [bug tracker][issues]
-
-## Config Validation
```

Comment:
I think it's worthwhile to leave this section in the README, even if it's redundant.